confmap/testdata/expand-escaped-env.yaml

```diff
@@ -0,0 +1,29 @@
+test_map:
+  # $$ -> escaped $
+  recv.1: "$$MAP_VALUE_1"
+  # $$$ -> escaped $ + $MAP_VALUE_2
+  recv.2: "$$$MAP_VALUE_2"
+  # $$$$ -> two escaped $
+  recv.3: "$$$$MAP_VALUE_3"
+  # $$$ -> escaped $ + substituted env var
+  recv.4: "$$${MAP_VALUE_2}"
```

Comment:
I'd like to see one like this but using the `${env:..}` syntax

Reply:
Added